Rakefile.rb

```diff
@@ -12,9 +13,14 @@
 
   desc 'Pull the integrations repo'
   task :'pull-integrations' do
+    integration_branch = ENV['VERSION'] || 'master'
+
+    sh "cd /#{ENV['INTEGRATIONS_REPO']} &&
```

Comment:
Maybe `Dir.chdir "/#{ENV['INTEGRATIONS_REPO']}"` instead ? But then we'd have to store the original repository.